content/doc/book/pipeline/syntax.adoc

```diff
@@ -248,6 +248,18 @@ need to contain its own `agent` directive. For example: `agent none`
 label:: Execute the Pipeline, or stage, on an agent available in the Jenkins
 environment with the provided label. For example: `agent { label 'my-defined-label' }`
 
+node:: Execute the Pipeline, or stage, on an agent available in the Jenkins
+environment with the provided label, with additional options. For example:
```

Comment:
Yes, `label` is required inside `node`. I'm legitimately not sure how we should call out `node` vs `label`, since they're just clones of each other in truth anyway, so if you've got a preferred wording you'd like me to add, I'm happy to do so.